apps/DataAggregator/appsettings.Development.json

```diff
@@ -22,15 +22,15 @@
                     "Name": "node1",
                     "CoreApiAddress": "http://localhost:3333",
                     "TrustWeighting": 1,
-                    "Enabled": true,
+                    "Enabled": true
                 },
                 {
                     "Name": "node2",
                     "CoreApiAddress": "http://localhost:3334",
                     "TrustWeighting": 1,
-                    "Enabled": false,
```

Comment:
Could we please rollback those JSON changes? We deliberately want those commas to reduce diff pollution.